aspnetcore/fundamentals/app-state/samples_snapshot/3.x/SessionSample/Startup.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Microsoft.AspNetCore.Builder;
```

Comment:
This file doesn't appear to be referenced anywhere. Just in case it _is_ needed, it's in a `3.x` folder but shows `2.x` code.